### PR TITLE
warpd: add page

### PR DESCRIPTION
**Pull request by zim0369**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
I think we can reorder the commands a bit. We could move **Line 30-32** (`hint mode`) to the top after `normal mode`.  Or we could move `normal mode` to last before `hint mode`.

**Review by CleanMachine1 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by zim0369** [comment]
@kbdharun I assumed that if the user tries these commands and bindings in order, then having them user the `--normal` mode would be a nice intro to how the cursor is used with warpd. Then in the end `--hint` mode would show them a convenient warpd feature.

**Comment by zim0369** [comment]
I would also like to know why listing multiple bindings in a single bullet isn't allowed by the parser. I felt that the movement bindings here would've been better in a single point:
```md
- Movement left/down/up/right:
`h`, `j`, `k` & `l` respectively
```

File: pages/linux/warpd.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`l`
+
+- Emulate left click:
+
+`m`
+
+- Run warpd in hint mode:
+
+`warpd --hint`
```
Comment (navarroaxel) [inline pages/linux/warpd.md:32]: We use to list examples about command arguments, then hot keys for interactive mode. So, IMO this should be the 2nd example in this page.

Comment (kbdharun) [reply pages/linux/warpd.md:32]: Agreed that was my suggestion too, as it would better align with our other pages. The PRs Author commented on this above.

**Pull request merged**